mne/viz/decoding.py

```diff
@@ -11,6 +11,127 @@
 import numpy as np
 import warnings
 
+def plot_topopatterns(csp, layout=None, vmin=None, vmax=None, sensors=True,
```

Comment:
`CSP.plot_patterns` for the method and `decoding.plot_csp_patterns` to pattern with other functions. just my 2c